crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -185,21 +185,11 @@ module sui::sui_system {
         p2p_address: vector<u8>,
         primary_address: vector<u8>,
         worker_address: vector<u8>,
-        stake: Coin<SUI>,
```

Comment:
Are we sure we don't want any lower bound on self-stake for non-genesis validators who are trying to join?